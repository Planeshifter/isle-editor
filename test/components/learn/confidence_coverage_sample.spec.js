// MODULES //

import React from 'react';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom/extend-expect';
import LearnConfidenceCoverageSample from '@isle-project/components/learn/confidence-coverage-sample/main.js';


// TESTS //

describe( '<LearnConfidenceCoverageSample />', function test() {
	it( 'renders a learning component', () => {
		const learning = renderer.create( <LearnConfidenceCoverageSample /> );
		expect( learning ).toMatchSnapshot();
	});
});
