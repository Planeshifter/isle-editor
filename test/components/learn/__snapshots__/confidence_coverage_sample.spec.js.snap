// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LearnConfidenceCoverageSample /> renders a learning component 1`] = `null`;
